docs/_ug/Troubleshooting.md

```diff
@@ -1,5 +1,13 @@
 <!-- markdownlint-disable-file first-line-h1 -->
+**Problem:** 
 
-Problem: JAR file not launching  even after double-clicking the file.
+JAR file not launching  even after double-clicking the file.
 
-Solution: On your terminal, run `java -jar "FILE_NAME.jar"`
+**Solution:**
+1. Open your terminal
+   - Windows: 
+     - The default key combination to launch your terminal is Ctrl + Shift + P
+   - Mac: 
+     - Use cmd + space to open Spotlight Search
+     - Search for "terminal" and open click it to launch.
```

Comment:
Should we add a note here to include the steps for Linux OS?

Reply:
I think people who use linux definitely know how to open a terminal tho.

Reply:
Let's use special GitHub-flavoured HTML `<kbd>` tags for this. Example:

```markdown
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
```

becomes <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>.

Reply:
Nice! Thanks for the suggestion! :D